app/views/sign_in/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+
+    <h1 class="govuk-heading-l">Ask for an account to register trainee teachers</h1>
+
+    <p class="govuk-body-m"> Although you have a DfE Sign-in account, you also need an account
+                             for this service.</p>
+
+    <p class="govuk-body-m"> Contact us at 
+    <%= govuk_mail_to "becomingateacher@digital.education.gov.uk", 
+                      "becomingateacher@digital.education.gov.uk" %> to ask for an account. 
+    </p>
+    
+    <%= govuk_link_to "Return Home", root_path, { class: "govuk-button" }  %>
```

Comment:
This isn't sufficient to fully make a button - it won't give it the correct semantic role or shim the behaviour.

[Apply has a helper](https://github.com/DFE-Digital/apply-for-teacher-training/blob/2780869dfcaddc9ae88f46c395e5ffca981fff66/app/helpers/view_helper.rb#L39) for this we could steal.

Also, please sentence case on the text `Return home`